### PR TITLE
crashpad 8da335ffad361191fa467de56a30715223f04014 (new formula)

### PR DESCRIPTION
**Pull request by shraddhaarora**
This is my first contribution for a formula, I'd appreciate any feedback you think is important.
(Made this a draft PR because it needs some changes)

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/c/crashpad.rb [hunk 1]
```diff
@@ -0,0 +1,181 @@
+class Crashpad < Formula
+  desc "Crash reporting system for client applications"
+  homepage "https://chromium.googlesource.com/crashpad/crashpad"
+  url "https://chromium.googlesource.com/crashpad/crashpad/+archive/8da335ffad361191fa467de56a30715223f04014.tar.gz"
+  version "8da335ffad361191fa467de56a30715223f04014"
```
Comment (SMillerDev) [inline Formula/c/crashpad.rb:5]: This does not seem to be a stable version, but instead just a random commit: https://docs.brew.sh/Acceptable-Formulae#stable-versions

Comment (shraddhaarora) [reply Formula/c/crashpad.rb:5]: I checked for version on their GitHub (https://github.com/chromium/crashpad), but it's not available there. I even scoured the repository for the same, and checked the commit history but no clue

Comment (jviotti) [reply Formula/c/crashpad.rb:5]: @SMillerDev Crashpad doesn't have any versioning strategy, actually. It comes from Chromium, where they just consume the tip of it. The only closest versioning strategy I can think of is checking what version a specific Chromium release uses and use that?

Comment (shraddhaarora) [reply Formula/c/crashpad.rb:5]: Chromium is currently using the same revision as mentioned in the PR

Ref: https://source.chromium.org/chromium/chromium/src/+/main:third_party/crashpad/README.chromium

Comment (SMillerDev) [reply Formula/c/crashpad.rb:5]: > Crashpad doesn't have any versioning strategy, actually. It comes from Chromium, where they just consume the tip of it.

Then it's not acceptable for Homebrew/core. 
Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

Comment (shraddhaarora) [reply Formula/c/crashpad.rb:5]: Oh okay, thank for sharing the information!

Comment (jviotti) [reply Formula/c/crashpad.rb:5]: @SMillerDev Would it be OK to use the corresponding Chromium versions instead, as they are the versions that govern all the dependencies as a mono repo? For example, if Chromium v1.2.3 uses a certain SHA of Crashpad, then set the Crashpad formula to v1.2.3? 

Alternatively, seems like Crashpad on VCPKG uses a timestamp instead: https://vcpkg.roundtrip.dev/ports/crashpad. 

Crashpad and its utilities are used by lots of people out there, so having it as an official formula would be great!

Comment (SMillerDev) [reply Formula/c/crashpad.rb:5]: It's not really a stable version from upstream though. The best way would really be for upstream to publish tags.

Comment (RaisinTen) [reply Formula/c/crashpad.rb:5]: If start maintaining a fork of crashpad and add version tags to it myself, would it be acceptable to create a formula for that?

Comment (RaisinTen) [reply Formula/c/crashpad.rb:5]: I had posted about this on the crashpad-dev google group and I have received a response from the Crashpad maintainers - https://groups.google.com/a/chromium.org/g/crashpad-dev/c/YABNQm6Hcss

Comment (shraddhaarora) [reply Formula/c/crashpad.rb:5]: > If I start maintaining a fork of crashpad and add version tags to it myself, would it be acceptable to create a formula for that?

@Moisan, could you help us with this?

Comment (SMillerDev) [reply Formula/c/crashpad.rb:5]: > If I start maintaining a fork of crashpad and add version tags to it myself, would it be acceptable to create a formula for that?

Seeing how upstream says that's a bad idea, and we specifically don't want to package forks.. that's not a solution

**Pull request closed without merging**